_schema.graphql

```diff
@@ -10925,7 +10934,7 @@ type Show implements Node {
   artworks(
     # List of artwork IDs to exclude from the response (irrespective of size)
     exclude: [String]
-    for_sale: Boolean
+    for_sale: Boolean = false
```

Comment:
Why the addition of this default?

Reply:
The dev had tried to add it, but used `default` as the key rather than `defaultValue`, so it was a bug, but no runtime change (as it would have been `null`).